### PR TITLE
Add tags used to smoothly turn on the self force

### PR DESCRIPTION
**Pull request by nikwit**
The tags will be used in a future PR.

### CODE REVIEW

File: src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.cpp [hunk 1]
```diff
@@ -29,12 +29,20 @@
...
-SelfForceOptions::SelfForceOptions(double mass_in, size_t iterations_in)
-    : mass(mass_in), iterations(iterations_in) {}
+SelfForceOptions::SelfForceOptions(const double mass_in,
+                                   const size_t iterations_in,
+                                   const double turn_on_time_in,
+                                   const double turn_on_interval_in)
+    : mass(mass_in),
+      iterations(iterations_in),
+      turn_on_time(turn_on_time_in),
+      turn_on_interval(turn_on_interval_in) {}
```
Comment (knelli2) [inline src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.cpp:39]: Is it ok if the turn on interval is zero? I believe it's in the denominator of the exponent

Comment (nikwit) [reply src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.cpp:39]: Yes, good catch. I limited it to 1M now which is still very low but can not cause any numerical issues.

File: src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.cpp [hunk 2]
```diff
@@ -36,7 +36,11 @@ SelfForceOptions::SelfForceOptions(const double mass_in,
     : mass(mass_in),
       iterations(iterations_in),
       turn_on_time(turn_on_time_in),
-      turn_on_interval(turn_on_interval_in) {}
+      turn_on_interval(turn_on_interval_in) {
+  if (turn_on_interval_in < 1.) {
+    ERROR("The turn on interval must be at least 1 M.");
+  }
```
Comment (knelli2) [inline src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.cpp]: The `lower_bound()` in the option in the `hpp` already takes care of this. No need to check here.

**Review by knelli2 — COMMENTED** [review]
Looks good, squash

**Comment by nikwit** [comment]
Done, thanks for the review!

**Review by knelli2 — APPROVED** [review]

**Pull request merged**